pymilvus/milvus_client/milvus_client.py

```diff
@@ -871,6 +871,21 @@ def create_schema(cls, **kwargs):
         kwargs["check_fields"] = False  # do not check fields for now
         return CollectionSchema([], **kwargs)
 
+    @classmethod
+    def create_field_schema(cls, name: str, dtype: DataType, desc: str, **kwargs) -> FieldSchema:
```

Comment:
Please use full name: data_type, description.

and please set `desccription` as optinonal by `descryption: str = ""`

Reply:
done